src/enigma2/Admin.cpp

```diff
@@ -19,6 +19,13 @@ using namespace enigma2::extract;
 using namespace enigma2::utilities;
 using json = nlohmann::json;
 
+Admin::Admin() : m_addonVersion(STR(VUPLUS_VERSION))
+{
+  std::string unknown = "unknown";
```

Comment:
Is it inteded to have this string not localized?

Reply:
Sorry, this will be overwritten, but should just be null terminated.